src/item/Raster.js

```diff
@@ -408,6 +408,8 @@ var Raster = Item.extend(/** @lends Raster# */{
             crossOrigin = this._crossOrigin;
         if (crossOrigin)
             image.crossOrigin = crossOrigin;
+        // We need to avoid setting image source to null, otherwise an unwanted
+        // error would be thrown in node context.
```

Comment:
perhaps better to mention that this is a JSDOM bug?

Reply:
How would you rephrase that as a good english speaker ? :)

Reply:
```
// Prevent setting image source to `null`, as this isn't supported by browsers,
// and it would actually throw exceptions in JSDOM.
//TODO: Look into fixing this exception in JSDOM